Dockerfile

```diff
@@ -18,6 +18,9 @@ RUN useradd freetz \
  && mkdir -p /freetz/images \
  && chown -R freetz /freetz
 
+RUN mkdir -p /patches \
+ && chown -R freetz /patches
+
```

Comment:
Only my preference: Could you add the commands to the last `RUN` so that no new layer is created. All of those commands run fast enough that Docker's build cache does not add any benefit.